src/common/imageio_module.h

```diff
@@ -231,6 +231,12 @@ int dt_imageio_get_index_of_storage(dt_imageio_module_storage_t *storage);
 /* add a module into the known module list */
 void dt_imageio_insert_storage(dt_imageio_module_storage_t *storage);
 
+// This function generates the "num" and "denum" fraction's elements to calculate the scaling factor
+// and improve the readability of the string itself displayed in the "scale" field of the
+// export settings from the string stored in the "plugins/lighttable/export/resizing_factor"
+// parameter of the configuration file.
+gchar *dt_imageio_resizing_factor_splitup(double *num, double *denum);
```

Comment:
Again, I asked to have the scale string as parameter here. I still think this is not ok, using an hidden pref as parameter for a public API sounds wrong.

Reply:
This function is quite specific and works with a single configuration parameter.
The current implementation made it possible to exclude an extra line of accessing the configuration from the code.
Your option returns it. But of course I can implement it.